frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-utilization.tsx

```diff
@@ -59,6 +77,10 @@ export const VMUtilizationCard: React.FC = () => {
     [vmName, namespace, launcherPodName],
   );
 
+  const adjustDuration = React.useMemo(() => {
+    return (start: number) => adjustDurationForVMI(start, vmi);
+  }, [vmi]);
```

Comment:
can we rather depend on `getCreationTimestamp(vmi)` rather then whole `vmi` object ? `vmi` object may change more often which will cause unnecessary recreation of the callback function

Reply:
done